netbox/extras/views.py

```diff
@@ -1059,10 +1059,6 @@ def get(self, request, module, name):
         report = module.reports[name]()
         jobs = module.get_jobs(report.class_name)
 
-        report.result = jobs.filter(
```

Comment:
`report.result` is referenced by the `report.html` template to display the time at which the report was most recently run.

Reply:
Sorry, I must have missed that one, putting it back in.